### ISSUE TITLE
Any way to get full screen background iframe?

### ISSUE THREAD

**Issue by ulfben** [OleVik/grav-plugin-presentation#4]
Reveal.js support background-iframe like this: 

```
<section data-background-iframe="https://example.net">
	<h2>Iframe</h2>
</section>
```

But I can't get that code to work in Grav + Presentation. At the moment my fallback is a regular  iframe:

```
---

<iframe class="embed-responsive" allowfullscreen="" mozallowfullscreen="" webkitallowfullscreen="" src="https://example.net"></iframe>

----
```

But of course, that means I can't overlay content, and the iframe doesn't fill the screen properly.

Your plugin already supports `[data-background-image=https://example.net]` - can we get a similar shortcode for [background-iframe](https://github.com/hakimel/reveal.js#iframe-backgrounds)?

**Comment by OleVik** [comment]
Sure, `[data-background-iframe=https://example.net]` is already supported, but I'll make it also add the `data-background-interactive`-attribute.

**Comment by ulfben** [comment]
Thank you! I was mistaken - I thought it didn't exist because it didn't work when first I tried it. I realized hours later that the problem was Privacy Badger (browser plugin) quietly blocking the load. At that point I had already tried six different ways of embedding content and completely forgot to retry the shortcode. :P

However, there does seem to be a bug with the shortcode. My actual URL contains a query string, and nothing after "?" is rendered in the front-end when using the shortcode. 

**Examples:**
`[data-background-iframe=https://example.net]` works perfectly and outputs 
`<section [...] data-background-iframe="https://example.net">`

But add some variables: 
`[data-background-iframe=https://example.net?h=f&sid=1006&p=0.m]` and you get: 
`<section [...] data-background-iframe="https://example.net?">`

Manually escape the variables:  
`[data-background-iframe=https://example.net?h=f&amp;sid=1006&amp;p=0.m]` and you still get: 
`<section [...] data-background-iframe="https://example.net?">`

I've tried wrapping the URL in quotes but that breaks the short code entirely. 

Manually writing an HTML iframe works fine with the query so it seems to be specific to the shortcode handling of URLs.

**Comment by OleVik** [comment]
Interaction was fixed in [v1.7.0-beta.1](https://github.com/OleVik/grav-plugin-presentation/releases/tag/v1.7.0-beta.1).
